### ISSUE TITLE
Add support for the Kinect

### ISSUE THREAD

**Issue by jspricke** [hcw70/OpenNI2#7]
Hi Hauke,

there is a fork of OpenNI2 with added Kinect support: https://github.com/piedar/OpenNI2-FreenectDriver. Maybe we should get that packaged as well.

Cheers Jochen

**Comment by hcw70** [comment]
Yes, that would be good. But maybe they should merge with upstream first? Else the debian/patches (where this might go) will explode.

Maybe we should have a debian package for the upstream devel branch, since there is the current development?

**Comment by hcw70** [comment]
Hi Jochen.

Had a quick look.

Since they merged from upstream/develop one day back in time, i tried to
merge within upstream/develop their stuff. It compiles ok. Cant test
however, since i dont have a kinect. So i propose we make a
openni2-snapshot debian package which tracks the upsteam/development
branch and see if they include the freenect branch there.

What do you think?

Regards

Hauke

**Comment by jspricke** [comment]
I've seen that you've contacted Ben and that there is a standalone version now, great :). I would propose to package it separately with dependencies on freenect and your OpenNI2 package.
